compiler/rustc_symbol_mangling/src/v0.rs

```diff
@@ -504,6 +504,7 @@ impl Printer<'tcx> for SymbolMangler<'tcx> {
 
         match ct.ty.kind() {
             ty::Uint(_) => {}
+            ty::Bool => {}
```

Comment:
If this is all that's required, can we also add `ty::Char` and `ty::Int`, or do these need special attention?

Reply:
ty::Int seems reasonable, though we might want to change mangling to express signed-ness (so you get `foo<-1>` in perf top or whatever, not `foo<255>`). I think we do not want to encode char with the bits so I would like to hold off on it for now.

Reply:
So both seem harder than just bool, which encoding with 0/1 seems quite reasonable.